src/components/app/shared/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import PandalCard from './PandalCard';
+import { TbLocationFilled } from 'react-icons/tb';
+import { IoMdTrendingUp } from 'react-icons/io';
+
+const Card = () => {
+  const [activeCard, setActiveCard] = useState('trending');
```

Comment:
add a generic here so people get autocomplete in future

`const [activeCard, setActiveCard] = useState<"trending"|"nearme">("trending");`

Reply:
okay okay